src/GoogleMaps.xml

```diff
@@ -65,6 +65,11 @@
             <description>Returns the list of locations to display</description>
             <returnType type="Object" entityProperty="locationsEntity" isList="true"/>
         </property>
+        <property key="autoZoom" type="boolean" defaultValue="true">
```

Comment:
Whether autozoom is true or false the zoom control will have to be shown if  it is enabled because auto zoom is executed only on first rendering